bioagents/mra/mra_module.py

```diff
@@ -412,8 +429,10 @@ def _get_matching_stmts(stmt_ref):
         kwargs = {}
     else:
         agent_list = []
-        kwargs = {k: v for k, v in zip(['subject', 'object'], agent_name_list)
-                  if v is not None}
+        kwargs = {k: v for k, v in zip(['subject', 'object'], agent_name_list)}
+        if not any(kwargs.values()):
+            raise BioagentException('Either subject or object must be '
+                                    'something other than None.')
```

Comment:
Is this Exception caught somewhere? We shouldn't fail just because supporting evidence is missing, the reply should still be a success.

Reply:
It is now :)